data/saas/config/slack_enterprise_config.yml

```diff
@@ -0,0 +1,57 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Slack Enterprise SaaS Config
+  type: slack_enterprise
+  description: A sample schema representing the Slack Enterprise connector for Fides
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+      default_value: slack.com
+    - name: user_oauth_token
+      label: User OAuth Token
+
+  client_config:
+    protocol: https
+    host: <domain>
+    authentication:
+      strategy: bearer
+      configuration:
+        token: <user_oauth_token>
```

Comment:
have we confirmed that using the oauth token as a static bearer token (that presumably needs to be manually rotated) is going to be acceptable for the initial users asking for this connector? understand the value of getting the other pieces of the connector set up and using this as a stop-gap, but just trying to understand if this is something we'll need to come back to, or if it's the path we want to support going forward